src/pages/MoviePages.css

```diff
@@ -0,0 +1,180 @@
+/* BACKGROUND IMAGE WITH GRADIENT */
+
+.info-backdrop {
+    width: 100%;
+    min-height: 100%;
+    position: fixed;
+    margin: -10px;
+    z-index: 1;
+    object-fit: cover;
+    display: block;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center top;
+    background-image: linear-gradient(to top, rgba(0, 0, 0, 0.9) 10%, rgba(255, 255, 255, 0) 80%), var(--img);
+    background-size: cover;
+}
+
+
+/* HOME BUTTON FUNCTIONALITY */
+
+.home {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 10px;
+    position: absolute;
+    z-index: 3;
+    font-size: 14px;
+    text-decoration: none;
+    padding-left: 15px;
+    padding-top: 15px;
+    text-shadow: 2px 1px 8px rgba(0, 0, 0, 0.548);
+}
+
+.home-button a {
+    text-decoration: none;
+    color: white;
+
+}
+
+.icon {
+    max-width: 30px;
+    filter: drop-shadow(3px 1px 5px rgba(0, 0, 0, 0.575));
+}
+
+.home-button:hover {
+    margin-left: -8px;
+    transition: 0.6s ease;
+}
+
+
+/* MOVIE INFO */
+
+.info-poster {
+    border: solid;
+    border-color: white;
+    border-width: 10px;
+    max-width: 180px;
+}
+
+.movie-info-box {
+    position: fixed;
```

Comment:
You probably meant `position: absolute;` here? You can't scroll to see all the text in mobile